solr/contrib/velocity/src/test/org/apache/solr/velocity/VelocityResponseWriterTest.java

```diff
@@ -176,14 +176,14 @@ public void testInitProps() throws Exception {
     // The test init properties file turns off being able to use $foreach.index (the implicit loop counter)
     // The foreach.vm template uses $!foreach.index, with ! suppressing the literal "$foreach.index" output
 
-    assertEquals("01", h.query(req("q","*:*", "wt","velocity",VelocityResponseWriter.TEMPLATE,"foreach")));
-    assertEquals("", h.query(req("q","*:*", "wt","velocityWithInitProps",VelocityResponseWriter.TEMPLATE,"foreach")));
+    assertEquals("01", h.query(req("q","*:*", "wt","velocity",VelocityResponseWriter.TEMPLATE,"foreach")).trim());
+    assertEquals("", h.query(req("q","*:*", "wt","velocityWithInitProps",VelocityResponseWriter.TEMPLATE,"foreach")).trim());
 
```

Comment:
For some reason these tests started failing due to an extra `NEWLINE` returned from `h.query()`. Have not tried to check whether this may be a problem for real templates or not, just fixed the failing tests. Anyone with an opinion here?